ac_c.lua

```diff
@@ -184,3 +205,35 @@ Citizen.CreateThread(function()
 		end
 	end
 end)
+
+function isCarBlacklisted(model)
+	for _, blacklistedCar in pairs(CarsBL) do
+		if model == GetHashKey(blacklistedCar) then
+			return true
+		end
+	end
+
+	return false
+end
+
+Citizen.CreateThread(function()
+	while true do
+		Citizen.Wait(500)
+		if IsPedInAnyVehicle(GetPlayerPed(-1)) then
+			v = GetVehiclePedIsIn(playerPed, false)
+		end
+		playerPed = GetPlayerPed(-1)
+		
+		if playerPed and v then
+			if GetPedInVehicleSeat(v, -1) == playerPed then
+				local car = GetVehiclePedIsIn(playerPed, false)
+				carModel = GetEntityModel(car)
+				carName = GetDisplayNameFromVehicleModel(carModel)
+				if isCarBlacklisted(carModel) then
+					Citizen.InvokeNative(0xAE3CBE5BF394C9C9, Citizen.PointerValueIntInitialized(car))
+					TriggerServerEvent('AntiCheese:CarFlag', carModel)
+				end
```

Comment:
i'd recommend also throwing them out of the vehicle or similar

Reply:
It removes the vehicle before warning

Reply:
i see, i recommend changing the InvokeNative to the actual Native name then, to prevent confusion